### ISSUE TITLE
totalDaysInMonth separate function for getting the total days in the given month

### ISSUE THREAD

**Issue by ohmmee** [techgaun/ad-bs-converter#7]
new Date(year, month, 0).getDate( ) can be used for english

similar type of inbuilt function for getting the last date of the month of BS date if there's any in the inner array of year and months data of every years.

The library gives the feature while conversion from AD to BS but not while passing BS
en: {
      totalDaysInMonth: totalDays
    }
is available but not for BS date like ('2075/05/06')
gotta do the hack like converting to AD, then getting totalDaysInMonth and if I'm working with only nepali months, parsing only year and month of BS and giving default value to date and the convert to AD and get totalDaysInMonth

Thankyou

**Comment by techgaun** [comment]
good point. thanks @ohmmee for opening this issue. this should not be a big lift. Would you be interested to tackle this and open a PR? Otherwise, I can find some time over next month or so to get this feature request in
